### PR TITLE
GHA: Automated transfer re-render with leanbuild update

### PR DESCRIPTION
**Pull request by cansavvy**
### Description:
 This PR has changes to this repository's files using the latest leanbuild package changes!
 If they don't look right, file an issue on the leanbuild package: https://github.com/jhudsl/leanbuild/issuesand do not merge.

### CODE REVIEW

File: manuscript/01-intro.md [hunk 1]
```diff
@@ -17,5 +17,6 @@ The course is intended for
 The curriculum will cover
 
 This course was funded by
+
```
Comment (cansavvy) [inline manuscript/01-intro.md:20]: Good! We wanted this extra line!

File: manuscript/topic-of-section.html [hunk 2]
```diff
@@ -314,12 +314,12 @@ <h2><span class="header-section-number">2.2</span> Print out session info</h2>
...
 ##  [1] knitr_1.33      hms_0.5.3       R6_2.4.1        rlang_0.4.10   
-##  [5] stringr_1.4.0   highr_0.8       httr_1.4.2      tools_4.0.2    
+##  [5] httr_1.4.2      stringr_1.4.0   highr_0.8       tools_4.0.2    
 ##  [9] xfun_0.26       htmltools_0.5.0 ellipsis_0.3.1  yaml_2.2.1     
 ## [13] leanbuild_0.1.2 digest_0.6.25   tibble_3.0.3    lifecycle_1.0.0
 ## [17] crayon_1.3.4    bookdown_0.24   readr_1.4.0     vctrs_0.3.4    
-## [21] curl_4.3        evaluate_0.14   rmarkdown_2.10  stringi_1.5.3  
-## [25] compiler_4.0.2  pillar_1.4.6    pkgconfig_2.0.3</code></pre>
+## [21] fs_1.5.0        curl_4.3        evaluate_0.14   rmarkdown_2.10 
+## [25] stringi_1.5.3   compiler_4.0.2  pillar_1.4.6    pkgconfig_2.0.3</code></pre>
```
Comment (cansavvy) [inline manuscript/topic-of-section.html:322]: Some packages got rearranged I think because I use `fs` now.

**Comment by cansavvy** [comment]
I would have expected to see the quiz copied over to manuscript here.

**Comment by carriewright11** [comment]
> I would have expected to see the quiz copied over to manuscript here.

me too...

**Comment by cansavvy** [comment]
I'll merge for now, because we do want the extra line. But I'll dive back into the copying issue.

**Pull request merged**